docs/auto-enable.md

```diff
@@ -0,0 +1,107 @@
+# Make `pg_stat_monitor` automatically accessible for every newly created database
+
+In PostgreSQL, a database is created by copying the [system template database](https://www.postgresql.org/docs/current/manage-ag-templatedbs.html) `template1`. To automatically create the `pg_stat_monitor` extension for every newly created database, you can do the following:
```

Comment:
I'm not sure I understand what you're trying to say in the first sentence... but here's what I _think_ you're going for:

```suggestion
In PostgreSQL, when a database is created the default settings of that database come from the [system template database](https://www.postgresql.org/docs/current/manage-ag-templatedbs.html) `template1`. To automatically create the `pg_stat_monitor` extension for every newly created database, you can do one of the following:
```